test/controllers/helpers/application.factory.ts

```diff
@@ -41,6 +42,10 @@ export const createApp = (configureBindings?: (container: Container) => void): A
     .build()
 }
 
+const mockEnvVars = (container: Container): void => {
```

Comment:
A lot of unit tests were failing without this, ones that I hadn't touched. Should be easy to extend this if similar issues arise again